l10n_it_ddt/wizard/ddt_from_picking.py

```diff
@@ -0,0 +1,134 @@
+        parcels = 0
+        for picking in self.picking_ids:
+            if picking.sale_id and picking.sale_id.parcels:
+                if parcels and parcels != picking.sale_id.parcels:
+                    raise UserError(
+                        _("Selected Pickings have different parcels"))
+                parcels = picking.sale_id.parcels
+                values['parcels'] = parcels
+        carriage_condition_id = False
+        for picking in self.picking_ids:
+            if picking.sale_id and picking.sale_id.carriage_condition_id:
+                if carriage_condition_id and (
+                        carriage_condition_id != (
+                            picking.sale_id.carriage_condition_id)):
+                    raise UserError(
+                        _("Selected Pickings have"
+                          " different carriage condition"))
+                carriage_condition_id = (
+                    picking.sale_id.carriage_condition_id)
+                values['carriage_condition_id'] = (
+                    carriage_condition_id.id)
+        goods_description_id = False
+        for picking in self.picking_ids:
+            if picking.sale_id and picking.sale_id.goods_description_id:
+                if goods_description_id and (
+                    goods_description_id != (
+                        picking.sale_id.goods_description_id)):
+                    raise UserError(
+                        _("Selected Pickings have "
+                          "different goods description"))
+                goods_description_id = picking.sale_id.goods_description_id
+                values['goods_description_id'] = (
+                    goods_description_id.id)
+        transportation_reason_id = False
+        for picking in self.picking_ids:
+            if picking.sale_id and (
+                    picking.sale_id.transportation_reason_id):
+                if transportation_reason_id and (
+                        transportation_reason_id != (
+                            picking.sale_id.transportation_reason_id)):
+                    raise UserError(
+                        _("Selected Pickings have"
+                            " different transportation reason"))
+                transportation_reason_id = (
+                    picking.sale_id.transportation_reason_id)
+                values['transportation_reason_id'] = (
+                    transportation_reason_id.id)
+        transportation_method_id = False
+        for picking in self.picking_ids:
+            if picking.sale_id and (
+                    picking.sale_id.transportation_method_id):
+                if transportation_method_id and (
+                    transportation_method_id != (
+                        picking.sale_id.transportation_method_id)):
+                    raise UserError(
+                        _("Selected Pickings have"
+                          " different transportation reason"))
```

Comment:
The error message is wrong: it should rather be "different transportation method"